modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/FullBlockInfoV1.scala

```diff
@@ -0,0 +1,40 @@
+package org.ergoplatform.explorer.http.api.v1.models
+
+import io.circe.Codec
+import io.circe.magnolia.derivation.decoder.semiauto.deriveMagnoliaDecoder
+import io.circe.magnolia.derivation.encoder.semiauto.deriveMagnoliaEncoder
+import org.ergoplatform.explorer.db.models.aggregates.{ExtendedAsset, ExtendedDataInput, ExtendedInput, ExtendedOutput}
+import org.ergoplatform.explorer.db.models.{Header, Transaction}
+import org.ergoplatform.explorer.http.api.v0.models.{HeaderInfo, TransactionInfo => TxInf}
+import sttp.tapir.{Schema, Validator}
+
+final case class FullBlockInfoV1(
```

Comment:
No need in V1 suffix, the class is already in api.v1 package.

Reply:
Models with same names are used in same files. Isn't this handier than import aliases ?